mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/init.jsp

```diff
@@ -18,6 +18,8 @@
 <%@ taglib uri="http://liferay.com/tld/aui" prefix="aui" %>
 <%@ taglib uri="http://liferay.com/tld/ui" prefix="liferay-ui" %>
 <%@ taglib uri="http://liferay.com/tld/theme" prefix="liferay-theme" %>
+<%@ page import = "java.io.*,java.util.*" %>
+<%@ page import="com.liferay.portal.kernel.language.UnicodeLanguageUtil" %>
```

Comment:
Your module dependencies in Intellij are not set, so we see an error here. Set dependencies for java version, development/global/portal libs, portal-kernel and util-taglib. See https://docs.google.com/document/d/1u_Xezzn0wlJUhNa88qWzwFMIBAmEq2OfEgoKW1ccuHo/edit#heading=h.ox5duuontk04, ignoring LCS modules setup.